sdk/src/main/java/com/ingresse/sdk/services/UserService.kt

```diff
@@ -50,6 +50,20 @@ class UserService(private val client: IngresseClient) {
      */
     fun cancelUpdateBasicInfos() = mUpdateBasicInfosCall?.cancel()
 
+    /**
+     * Method to cancel user tickets data request
+     */
+    fun cancelUserTicketsData(concurrent: Boolean = false) {
+        if(!concurrent) {
+            mUserTicketsCall?.cancel()
```

Comment:
I think it's better make a cancelUserTicketCall and cancelUserTransfersCall separated. Remember that SDK idea it's to serve others, so they maybe will not use both calls together.